commands/intensify.js

```diff
@@ -0,0 +1,74 @@
+const Command = require('./command.js')
+const Image = require('../models/image.js');
+const ImageLoader = require('../util/image-loader.js')
+const ImageUploader = require('../util/image-uploader.js')
+const UtilArray = require('../util/array.js')
+const {TRANSPARENT_BLACK} = require('../constants.js')
+
+class Intensify extends Command {
+    static help() {
+        return `*Usage:* \`emojitools intensify <url>\``
+    }
+
+    async render() {
+        const image = await ImageLoader.fromUrl(this.getUrl());
+        const frame_delay = 5;
```

Comment:
general style nits: please use camelCase rather than snake_case, remove semicolons, and access object properties using dot notation (`image.frames` rather than `image['frames']`)